ext/cl_khr_tensor.asciidoc

```diff
@@ -0,0 +1,547 @@
+// Copyright 2023 The Khronos Group. This work is licensed under a
+// Creative Commons Attribution 4.0 International License; see
+// http://creativecommons.org/licenses/by/4.0/
+= cl_khr_tensor
+
+:source-highlighter: coreray
+
+[[cl_khr_tensor]]
+== Tensor Data Type
+
+Purpose of this extension is to provide ...
```

Comment:
"This extension provides a new opaque OpenCL datatype called `cl_tensor`. It is used for storing N-dimensional tensor data in implementation-optimized way. The datatype is designed to be efficiently used within the `cl_khr_command_buffers` extension to capture task graphs which can utilize tensors as input, output and temporary storage."  [etc etc]

Reply:
Added this and tweaked it a little.